index.js

```diff
@@ -143,8 +143,13 @@ async function executeAction() {
     let status = run_status[res.data.status];
     core.info(`Run: ${res.data.id} - ${status}`);
 
-    if (res.data.is_complete) {
-      core.info(`job finished with '${status}'`);
+    if (core.getBooleanInput('wait_for_job')) {
+      if (res.data.is_complete) {
+        core.info(`job finished with '${status}'`);
+        break;
+      }
+    } else {
+      core.info("Not waiting for job to finish. Relevant run logs will be omitted.")
```

Comment:
This does work, but what about *not* going into the `while (true)` at all and just skipping it?

```js
if (not core.getBooleanInput('wait_for_job')) {
      core.info("Not waiting for job to finish. Relevant run logs will be omitted.")
      return
}
while (true) {
  ...
```

Reply:
The main downside would be that `res` won't be populated, so the `git_sha` output won't work. But totally open to whatever approach you think best!

Reply:
Oh, this is a great point! I think it's good then.